R/convertFlowCore.R

```diff
@@ -35,3 +35,37 @@ convertGate <- function (gate) {
     }
   )
 }
+
+#' Scale flowCore gate
+#'
+#' Applies a scale to points from a flowCore gate, so that it may
+#' be saved to CellEngine.
+#'
+#' @param gate The flowCore gate points (i.e. PologonalGate@boundaries)
+#' @param scaleSet The CellEngine ScaleSet.
+#' @export
+#' @examples
+#' \dontrun{
+#' sngl <- flowDensity(file, params)
+#' flowGate <- sngl@filter
+#' scaleSet <- getScaleSets("5d2f8b4b21fd0676fb3a6a8c")
+#' gatePoints <- scaleFlowCoreGate(gate, scaleSet)
+#' createPolygonGate(id, colnames(gate)[1], colnames(gate)[2], 'my gate', vertices=gate_points)
```

Comment:
It'd be nice to have a `convertFromFlowCore` function that does this.

Reply:
I'm a little wary of that for two reasons:
1) the gate to convert isn't always a `flowDensity` object
2) the function makes several operations, including the API call to CE, which is more involved than what `convertToFlowCore` does.

That said, I've implemented it; if these aren't real concerns for you, then great :+1:

Reply:
I've exported `scaleFlowCoreGate` as well as `convertFromFlowCore`; thoughts?

Reply:
See last comment in the new review. I think it looks good. (1) I'm proposing to make it generic to avoid this issue. (2) The way you wrote it, users can avoid that API call by retrieving the scaleset once.